apps/src/codebridge/FilePreview/InnerHTMLPreview.tsx

```diff
@@ -1,8 +1,9 @@
 import {DEFAULT_FOLDER_ID} from '@codebridge/constants';
-import {createBlobUrlForFile, getFolderPath} from '@codebridge/utils';
+import {getUrlForFile, getFolderPath} from '@codebridge/utils';
 import React, {useCallback, useEffect, useMemo, useRef, useState} from 'react';
 
 import {MultiFileSource} from '@cdo/apps/lab2/types';
+import {WEBLAB2_IMAGE_FILE_TYPES} from '@cdo/apps/weblab2/constants';
```

Comment:
Grabbing weblab2 constants in this directory gave me a little code smell? Maybe we should move this file to the weblab2 dir?

Reply:
fine with me!